package.json

```diff
@@ -12,30 +12,31 @@
   },
   "workspaces": [
     "apps/**/*",
-    "packages/**/*"
+    "packages/**/*",
+    "./"
   ],
   "scripts": {
-    "build": "turbo run build",
+    "build": "turbo run build --filter=!pie-monorepo",
```

Comment:
what does this do? :)

Reply:
I'm guessing it runs it for all packages except the root monorepo package?

Reply:
Correct. If we don't provide this, turborepo goes into a recursive loop 😭